### ISSUE TITLE
Only works if your list container is a div.

### ISSUE THREAD

**Issue by simondrabble** [jackrabbitsgroup/angular-infinitescroll#3]
Since the scroll container gets converted into a DIV, any contained elements could be invalid, e.g. I have a container that is a UL element with ng-repeat on the contained LI. The parent (UL) gets converted into a DIV, so now all of those generated LI elements have lost the correct structure.

**Comment by lukemadera** [comment]
Fair point; I've only tried with div's. It may be a non-trivial change but pull requests are welcome!
